README.md

```diff
@@ -1 +1,62 @@
-# -Global-Warming
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Global Warming / Gapstone one
+
+> Global Warming responsive website.
+> This version contain main and about page.
+> The next version will contains more pages.
+## Built With
+
+- HTML&CSS
+- VScode 
+- JavaScript
+
+## Live Demo 
+
+Go to live demo : 
+
+![Global Warming](https://drive.google.com/file/d/1t-vB2QbthrtTBqBV6YxEUTHenEgFLf01/view?usp=sharing)
```

Comment:
Kindly deploy the project using GitHub pages and add the link in the README file as it's in the requirement of the project.